src/api.ts

```diff
@@ -348,6 +365,24 @@ class NmdcServerClient extends FetchClient {
     });
   }
 
+  async getSubmissionLock(id: string) {
```

Comment:
This is one situation where I think `get...` is more ambiguous than I would like. I've gotten used to `get...` being like "read"—as opposed to "claim". I'm wondering whether using a synonym/alternative here would help highlight that we are doing something other than _reading_ here. Ideas:
- `claim`
- `acquire` (synonym of `get`, but breaks me free of assumptions about `get`)
- `obtain` (same as above)

To clarify, I don't consider `get...` to be a dealbreaker.

Reply:
I think that's a good idea. Especially because it's literally not a `GET` request. I think in the comments I already mostly used the word "acquire".